lib/rules/block-indentation.js

```diff
@@ -45,7 +45,9 @@
  * boolean -- `true` indicates a 2 space indent, `false` indicates that the rule is disabled.
  * numeric -- the number of spaces to require for indentation
  * "tab" -- To indicate tab style indentation (1 char)
-
+ * object --
+ *    `indentation: <numeric>` - number of spaces to indent',
```

Comment:
Can you mention add `(default 2)` to this line?

Reply:
Updated